src/extension/appLauncher.ts

```diff
@@ -220,6 +225,35 @@ export class AppLauncher {
                                 this.logger.info(localize("PrewarmingBundleCache", "Prewarming bundle cache. This may take a while ..."));
                                 return mobilePlatform.prewarmBundleCache();
                             })
+                            .then(() => {
+                                if (launchArgs.platform === "android" && mobilePlatform instanceof AndroidPlatform) {
+                                    return mobilePlatform.startEmulator(launchArgs.target);
+                                }
+                                else return null;
+                            })
+                            .then((emulator: IEmulator | null) => {
+                                if (emulator) {
+                                    let launchConfigIndex = this.launchScenarosManager.getFirstScenarioIndexByParams(launchArgs);
+                                    const launchScenarios = this.launchScenarosManager.getLaunchScenarios();
+                                    if (launchConfigIndex !== null && launchConfigIndex !== undefined && launchScenarios.configurations) {
```

Comment:
Is it important for us to use strict comparison with `null` and `undefined ` here? Can we just replace it with such construction:
```
if (launchConfigIndex && launchScenarios.configurations) 
```